### PR TITLE
Basic admin panel

### PR DESCRIPTION
**Pull request by jeppes**

### CODE REVIEW

File: client/src/components/admin/Admin.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    }
+    this.startAddingBuilding = this.startAddingBuilding.bind(this)
+    this.updateNewBuilding = this.updateNewBuilding.bind(this)
+    this.addBuilding = this.addBuilding.bind(this)
+    this.cancelBuilding = this.cancelBuilding.bind(this)
+    this.renderBuilding = this.renderBuilding.bind(this)
+  }
+
+  componentDidMount() {
+    database.ref('/buildings').on('value', snapshot =>{
```
Comment (mictab) [inline client/src/components/admin/Admin.js]: Can we add a space here after =>?

File: client/src/components/admin/Admin.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      this.setState({ buildings: toList(snapshot.val()) })
+    }
+    )
```
Comment (mictab) [inline client/src/components/admin/Admin.js]: Put this on the previous line?

File: client/src/components/admin/Admin.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    return (
+      <div key={name}
+        className='admin-building'>
+        <span
+          className={'admin-building-header'}>
+          {name}
+        </span>
+        <span
+          className={'admin-building-count ' + overcrowdedClass}>
+          ({count}/{limit} people) { isOvercrowded && 'WARNING: OVERCROWDED' }
```
Comment (mictab) [inline client/src/components/admin/Admin.js:90]: We're gonna need something like this in the counter component as well :)

File: client/src/components/admin/Admin.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            ) : (
+              <button
+                onClick={this.startAddingBuilding}>
+                Add building
+              </button>
+            )
+          }
+        </div>
+        <div>
+
```
Comment (mictab) [inline client/src/components/admin/Admin.js]: Remove extra line

**Review by mictab — APPROVED** [review]

File: client/src/components/admin/Admin.js (hunk 3 above)
Comment (jeppes) [reply client/src/components/admin/Admin.js:90]: 👍

**Pull request merged**